cms/config.js

```diff
@@ -19,7 +19,7 @@ export default {
       label_singular: 'Service',
       name: 'service',
       folder: 'content/services',
-      slug: '{{slug}}',
+      slug: '{{organisation}}-{{title}}',
```

Comment:
I've forgotten how the slug field works. Reassure me that this will result in a fully slugified string?

Reply:
I think so, it's similar to the example here using field names https://www.netlifycms.org/docs/configuration-options/#slug
I'll do a quick test.